src/Oro/Bundle/ConfigBundle/Resources/config/system_configuration.yml

```diff
@@ -7,6 +7,9 @@ oro_system_configuration:
         general_setup:
             title: oro.config.system_configuration.groups.general_setup.title
 
+        integration_setup:
```

Comment:
Because I noticed you have basic group (general_setup) inside ConfigBundle.
integration_setup seems to be another general group which could use more bundles than UserBundle.

So I thought ConfigBundle is the place. But I'll move it. (the same for massages below).